### ISSUE TITLE
NVDA is not reading icons when hamburger menu is collapsed

### ISSUE THREAD

**Issue by bhavya974** [nvaccess/nvda#9099]
### Steps to reproduce:
1. Open the attached link  below in Chrome Browser.
   link: https://www.w3schools.com/code/tryit.asp?filename=FYDE0Z95TXNK
2. Start the NVDA and run the code.
3. Go through the icons, you will observe that NVDA doesn't read anything about the icons.

### Actual behavior:
NVDA is not reading icons when hamburger menu is collapsed

### Expected behavior:
NVDA should read the icon names correctly when hamburger menu is in collapsed state as well

#### Name and version of other software in use when reproducing the issue:
Chrome- Version 70.0.3538.110 (Official Build) (64-bit)

#### Does the issue still occur after restarting your PC?
Yes, it does.

#### Have you tried any other versions of NVDA?
No, I didn't.

**Comment by leonardder** [comment]
What makes you believe that this behaviour is caused by NVDA and not due to wrong web authoring?

**Comment by josephsl** [comment]
Hi, how about with JAWS, as well as with other web browsers? Thanks.

**Comment by bhavya974** [comment]
I checked with JAWS in both edge and chrome. In both the browsers it's working fine. Didn't check with any other browsers.

**Comment by Adriani90** [comment]
I confirm this does not work in NVDA. This does not work in any browser. I tested also in Firefox 72,0.2 and IE, is not working. Narator reads the menu items here.
cc: @jcsteh do you have any suggestions at least for Firefox?

**Comment by jcsteh** [comment]
From what I can see, this is a tree control. The expected behaviour for a tree control in browse mode is that only the selected item is read, as is also the case for a list box. It's expected that you navigate tree controls in focus mode, since tree controls may contain many items and are also hierarchical.

**Comment by Adriani90** [comment]
Actually I am not sure what exactly is expected here:
In browse mode, NVDA does not report selected item, in fact I think it is not possible to select anything.
In focus mode and Firefox, NVDA can navigate the icon items by pressing tab, but not by pressing arrow keys in the tree view.
In Chrome and Edge, NVDA cannot access the icons in focus mode at all, but this is also valid for Narator and Jaws.

Narator is the only screen reader that can work with the icons both in scan mode and in focus mode.

I think the implementation and rendering of this kind of tree view is some how buggy or not thought to the end yet.

cc: @tspivey, @stevefaulkner, @scottaohara.

**Comment by scottaohara** [comment]
the markup pattern isn't particularly well formed, and being only HTML is not functional at all - a list is included within a treeitem, for instance, and the only icon i'm noticing is broken in its rendering, marked with both role=presentation and an aria-label that is the same as it's parent treeitem's aria-label.... so, it's rather unclear what NVDA issue is here, if any at all.

**Comment by Adriani90** [comment]
Thank you very much for this updated explanation, I think it makes sense to close this as invalid due to invalid web authoring. At this stage it is not possible to investigate this issue effectively. 

@bhavya974 please provide an updated minimal test case with a valid markup patern and we can reopen then.
